<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="mk.ukim.finki.SILab2Test" tests="5" skipped="0" failures="0" errors="0" timestamp="2023-05-29T13:09:44" hostname="DESKTOP-9RRSD5S" time="0.043">
  <properties/>
  <testcase name="test2()" classname="mk.ukim.finki.SILab2Test" time="0.031"/>
  <testcase name="test3()" classname="mk.ukim.finki.SILab2Test" time="0.002"/>
  <testcase name="test4()" classname="mk.ukim.finki.SILab2Test" time="0.001"/>
  <testcase name="test5()" classname="mk.ukim.finki.SILab2Test" time="0.001"/>
  <testcase name="exceptionTest()" classname="mk.ukim.finki.SILab2Test" time="0.004"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
